py/selenium/webdriver/common/selenium_manager.py

```diff
@@ -53,6 +54,10 @@ def get_binary() -> Path:
 
         path = Path(__file__).parent.joinpath(directory, file)
 
```

Comment:
Can we add a test for this or this is just generic enough?

Reply:
What kind of test? The code itself tests that the path exists on the next few lines, so I'm not sure what else we'd test for.

Reply:
It's fine, don't worry about it